### DISCUSSION TITLE
Hitcircle showing on sliderend

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#16883] [Q&A] [answered]
I tested this on a few replays, but this only happens on mine where is miss that exact slider.  

Beatmap: https://osu.ppy.sh/beatmapsets/1533832#osu/3136646
Replay file: https://pek.li/omxbnq.osr (GitHub didn't let me upload that file type)

https://user-images.githubusercontent.com/22381687/154198911-6403e52c-7312-4674-9c54-f3c893c71fd1.mp4

**Comment by peppy** [comment, marked as answer]
Pretty sure that's expected behaviour with snaking turned on. The head will follow the slider until it's hit.

**Reply by respektive** [reply]
Hmm, that actually makes sense. Somehow didn't think of that. I thought the head would disappear either way and just the body would move. 😄
